service/s3/s3crypto/s3crypto_client.go

```diff
@@ -0,0 +1,217 @@
+package s3crypto
+
+// DR TOOLS TEST integration test bucket
+import (
+	"bytes"
+	"encoding/base64"
+	"io/ioutil"
+	"math/rand"
+
+	"github.com/aws/aws-sdk-go/aws/client"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+const (
+	ivSize  = 16
```

Comment:
Yea, that was for specifically `cbc`, that's going to change with the comment above.